pkg/controllers/clusterresourceplacement/resource_selector.go

```diff
@@ -120,10 +132,10 @@ func (r *Reconciler) fetchClusterScopedResources(_ context.Context, selector fle
 		Kind:    selector.Kind,
 	}
 	if !r.InformerManager.IsClusterScopedResources(gvk) {
-		return nil, fmt.Errorf("%+v is not a cluster scoped resource", restMapping.Resource)
+		return nil, controller.NewUserError(fmt.Errorf("%+v is not a cluster scoped resource", restMapping.Resource))
 	}
 	if !r.InformerManager.IsInformerSynced(gvr) {
-		return nil, fmt.Errorf("informer cache for %+v is not synced yet", restMapping.Resource)
+		return nil, controller.NewExpectedBehaviorError(fmt.Errorf("informer cache for %+v is not synced yet", restMapping.Resource))
```

Comment:
This is an example of my confusion specified in an earlier comment. It could be, at this point, that the cache is simply not working fast enough and will get sync'd soon; or it could also be that there is a connection/RBAC issue that will never recover without intervention and needs attention.